src/main/java/io/appform/dropwizard/discovery/bundle/resolvers/CriteriaResolver.java

```diff
@@ -0,0 +1,7 @@
+package io.appform.dropwizard.discovery.bundle.resolvers;
+
+public interface CriteriaResolver<T, A> {
+
+  T getValue(A arg);
```

Comment:
rename to buildNode() or createNode() or just node()

Reply:
This is an interface and doesn't understand node. Best I can do is rename it to resolve

Reply:
Added a comment on the MR, removing the generic altogether.